.github/workflows/release_nightly.yml

```diff
@@ -129,6 +136,15 @@ jobs:
           cp README.md package/README.md
           cp LICENSE.md package/LICENSE.md
 
+      - name: Package MSI
+        run: |
+          cd desktop/wix
+          wix build ruffle.wxs -ext WixToolset.UI.wixext -ext WixToolset.Util.wixext -arch x64 -o ../../package/setup.msi -pdbtype none
+        env:
+          RUFFLE_VERSION: "0.1.0"
```

Comment:
Can we use version from cargo here? (see https://github.com/ruffle-rs/ruffle/pull/17193/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R93)

Reply:
Good idea, yes we can!

But let's do that after your PR, since we're not using `cargo get` yet. I left it here to easily show future-us how to hook it up.